_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -415,6 +415,35 @@ public static function register_endpoints() {
 				),
 			)
 		) );
+
+		// Get and set API keys.
+		register_rest_route(
+			'jetpack/v4',
+			'/api-key/(?P<service>[a-z\-_]+)',
```

Comment:
I suggest `/api-keys/` (plural). Even if there is no list endpoint, we're still talking about a collection (of API keys), so it should be plural (like the posts, comments, etc. endpoints).

Reply:
Combined this with a comment from @enejb and opted for `/service-api-keys/` to be fully clear about the purpose of the endpoint.